projects/UConn_Zhu.yaml

```
@@ -0,0 +1,10 @@
+Description: GERS Laboratory working on big earth observation
+  data and environmental change
```

```suggestion
Description: >
  GERS Laboratory working on big earth observation
  data and environmental change
```

Use YAML line continuations